### ISSUE TITLE
Add Buster to available Debian versions

### ISSUE THREAD

**Issue by FabGre** [dj-wasabi/ansible-zabbix-server#170]
**Is your feature request related to a problem? Please describe.**

I try to install Zabbix Server on a Debian Buster server but it's not available for now : 

```
TASK [dj-wasabi.zabbix-web : Debian | Install gpg key] ********************************************************************************************************************************
fatal: [-snip-]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'buster'\n\nThe error appears to be in '/-snip-/external/dj-wasabi.zabbix-web/tasks/Debian.yml': line 10, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"Debian | Install gpg key\"\n  ^ here\n"}
```

**Describe the solution you'd like**

I would like to have Debian Buster available. Did you planned to add it soon ?

Thanks in advance and thanks for your work.

**Comment by dj-wasabi** [comment]
What Zabbix Server are you trying to install? Buster is available for Zabbix >=4.0

**Comment by dj-wasabi** [comment]
Oh, you are installing the web. Different repo.
